account_invoice_pricelist_technical/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module extends the functionality of the Technical pricelist Module (same repository) to prevent user to select undesired pricelist on account invoice form.
```

Comment:
```suggestion
This module extends the functionality of the Technical Pricelist module (same repository) to prevent users from selecting technical pricelists on account invoice forms.
```